android/engine/src/main/java/org/smartregister/fhircore/engine/ui/usersetting/UserSettingViewModel.kt

```diff
@@ -116,4 +129,9 @@ constructor(
       accountAuthenticator.launchScreen(AppSettingActivity::class.java)
     }
   }
+
+  fun isP2PSyncAvailable(): Boolean {
+    val p2pMenu = (navigationConfiguration.staticMenu.find { it.id == NAVIGATION_KEY_P2P })
+    return p2pMenu?.visible ?: false
+  }
 }
```

Comment:
Use the [ApplicationConfiguration.deviceToDeviceSync](https://github.com/opensrp/fhircore/blob/4d0c0abfc8216816d33bde1a4ea0cbacaaf5c64c/android/engine/src/main/java/org/smartregister/fhircore/engine/configuration/app/ApplicationConfiguration.kt#L34) instead, like we discussed if the property is not null then p2p is enabled, disabled otherwise. We cannot rely on the navigation config.